src/platform_impl/linux/window.rs

```diff
@@ -743,7 +749,7 @@ pub enum WindowRequest {
   CursorIcon(Option<CursorIcon>),
   CursorPosition((i32, i32)),
   CursorIgnoreEvents(bool),
-  WireUpEvents,
+  WireUpEvents(bool),
```

Comment:
Should document what is the meaning of this `bool` or use an enum struct variant.